getsolus/packages
# Add ent to repo and use it for taskfile

**Summary**

- ent: Add at v0.1.0
- Taskfile.yml: Add new command to check package updates

**Test Plan**


- Check updates on packages repo
- Use `go-task updatecheck`

**Checklist**
